Examples/SnapinsSDKKotlinExample/app/src/javaImplDebug/java/com/salesforce/snapinssdkexample/activities/settings/ChatSettingsActivity.java

```diff
@@ -0,0 +1,38 @@
+package com.salesforce.snapinssdkexample.activities.settings;
+
+import android.os.Bundle;
+import android.preference.PreferenceFragment;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import com.salesforce.snapinssdkexample.R;
+
+
+/**
+ * Activity to display settings for live agent chat.
+ */
+
+public class ChatSettingsActivity extends AppCompatActivity {
+    public static final String KEY_ORG_ID = "pref_chat_org_id";
+    public static final String KEY_DEVELPMENT_ID = "pref_chat_deployment_id";
+    public static final String KEY_BUTTON_ID = "pref_chat_button_id";
+    public static final String KEY_LIVEAGENT_POD = "pref_chat_la_pod";
```

Comment:
nit: `KEY_LIVEAGENT_POD` -> `KEY_LIVE_AGENT_POD` to match the `kotlinImplDebug` variant.

Reply:
done